data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,472 @@ mission "Unfettered returning home"
+			`	He then looks at you intensively, straight in the eyes, with a sudden change of tone. "We sometimes bend the rules for our friends, but that is not the case with you".`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	Mera starts to look a bit impatient. "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships, but Mera pulls you in close once they are far enough away. "I want this done by the end of the day, so do not try my patience. You are lucky I offered this deal in the first place, as they were planning to take the relic by force. Now, quick, get to your ship!"`
+			`	You see the Unfettered have already begun their launching procedures, so you should prepare for takeoff in your Anomalocaris, and no other ships, as soon as you can.`
+				accept
+			label headingout
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				set "unfettered: fled with the relic"
+			`	Seeing as they are unlikely to cooperate, you quickly head to your ship and launch, with the Unfettered rushing after you shortly after.`
+				launch
+	on enter
+		conversation
+			branch fleeing
+				has "unfettered: fled with the relic"
+			branch "too many ships"
+				"total ships" > 1
+			`As you leave the planet's atmosphere, you see the challengers already waiting for you in the system. They hail you, "Luck has nothing to do with it. Show us what you got, monkey! Show us that you control your weapons well, as any warriors should. Prove to us that you have the strength to disable our ship, yet still have the restraint not to destroy them."`
+				accept
+			label fleeing
+			action
+				fail
+			`The Unfettered don't seem too pleased to see you leaving with their prize. The challengers are already waiting for you in orbit, and hail you. "Where do you think you're going, monkey?"`
+				decline
+			label "too many ships"
+			action
+				fail
+			`As you take to join your fleet in orbit, You receive a hail from the Unfettered. "If you can't even count how many ships you have you're not worthy of our time, monkey!"`
+	on disabled
+		fail
+		conversation
+			`The enemy ships hail you. "Thought you could get away from us so easily? You better bribe us and then land on our worlds quick, before we value blowing you up more than preserving that ship!"`
+				to display
+					has "unfettered: fled with the relic"
+				decline
+			`One of the enemy ships hails you. "You have failed the challenge. We will take the ship from you when we see fit. I'll fix your ship and leave you alone for a bribe, but I suggest you land on an Unfettered world and return the ship to us, with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
+			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us, but maybe someday you will prove yourself otherwise."`
```

Comment:
```suggestion
			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us. Maybe someday you will prove yourself otherwise."`
```
I feel this reads better, but I'm not married to it.